.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: python
+python:
+- 3.6
+before_install:
+- touch .env
```

Comment:
This is a placeholder. I suggest using a new API key just for Travis and adding it using `travis encrypt TOKEN="123456789" --add`